.github/workflows/CI.yml

```diff
@@ -14,9 +11,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.3'  # oldest Julia version that works
-          - '1.8'  # newest Julia version that works with ModelingToolkit
-          - '1'
+          - '1.10'
```

Comment:
I would not recommend removing the tests on older Julia versions and only testing on v1.10. It is very easy to introduce changes that break old versions unnoticed (I saw it happening many times), particularly because you presumably do not use them for development anymore. If v1.10 is the lowest supported version, that should be reflected in Project.toml (see the other comment).

Reply:
I'm inclined to just set the lower bound as 1.10 since that's the new LTS version, right?

Reply:
Totally reasonable.
But note that v1.6-v1.9 would still be installing the current release. So if it broke, that will leave the package in a broken state for these Julia versions. (You could retroactively fix the entry in the registry to raise the Julia version in that case.)